src/content/app/genome-browser/components/track-settings-panel/track-settings-views/GeneTrackSettings.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { useAppSelector, type RootState } from 'src/store';
+
+import SlideToggle from 'src/shared/components/slide-toggle/SlideToggle';
+import GlobalTrackSwitch from './components/global-track-switch/GlobalTrackSwitch';
+
+import { type GeneTrackSettings as GeneTrackSettingsType } from 'src/content/app/genome-browser/state/track-settings/trackSettingsSlice';
+import {
+  getApplyToAllSettings,
+  getBrowserSelectedCog,
+  getTrackSettingsForTrackId
+} from 'src/content/app/genome-browser/state/track-settings/trackSettingsSelectors';
+
+import useTrackSettings from '../useTrackSettings';
+
+import styles from '../TrackSettingsPanel.scss';
+
+export const GeneTrackSettings = () => {
+  const selectedCog = useAppSelector(getBrowserSelectedCog) || '';
+  const selectedTrackSettings = useAppSelector(
+    (state: RootState) =>
```

Comment:
You don't need to declare the type of `state` here — `useAppSelector` will already know it.